examples/typescript/package.json

```diff
@@ -4,10 +4,9 @@
     "test": "eslint ."
   },
   "devDependencies": {
-    "@eslint/js": "^8.56.0",
-    "eslint": "^8.56.0",
-    "eslint-plugin-markdown": "file:../..",
+    "@eslint/js": "^9.7.0",
+    "eslint": "^9.7.0",
     "typescript": "^5.3.3",
-    "typescript-eslint": "^7.0.1"
+    "typescript-eslint": "8.0.0-alpha.51"
```

Comment:
This is the latest prerelease. We'll need to use a prerelease version because v7 doesn't support ESLint 9. When final v8 is released, this should be changed to `"^8.0.0"`.